tools/Makefile.unix

```diff
@@ -313,6 +313,7 @@ ifneq ($(CONFIG_ARCH_CHIP),)
 endif
 
 dirlinks: include/arch include/arch/board include/arch/chip $(ARCH_SRC)/board $(ARCH_SRC)/chip drivers/platform
+	$(Q) $(MAKE) -C libs/libxx dirlinks TOPDIR="$(TOPDIR)"
```

Comment:
This should go to Makefile.win too.

Reply:
Good catch, I will provide a patch fix it.

Reply:
@Ouss4 done.